media/src/App.svelte

```diff
@@ -387,6 +384,16 @@
                     val.uuid, val.kind, val.params, objects);
             }
         }
+
+        const inputarea = document.getElementsByClassName('objectBoxOuter')[0];
+        inputarea.addEventListener("focusin", () => {
+            controls.enabled = false;
+            controls2.enabled = false;
+        })
+        canvas.addEventListener("click", () => {
```

Comment:
It seems this `click` listener should instead be the `focusout` event here.

With the proposed change, it's possible to permanently disable keyboard controls if you're not using the mouse, by triggering `focusout` via keyboard tabbing. In general we don't want to explicitly listen for `click` events for core functionality, as it makes things less accessible. `focusout` would be the more general case here, as it can be triggered via keyboard or mouse - the input doesn't matter.

Reply:
Excellent point